Doc/library/zipfile.rst

```diff
@@ -385,10 +385,11 @@ ZipFile Objects
 
 .. method:: ZipFile.writestr(zinfo_or_arcname, data[, compress_type])
 
-   Write the string *data* to the archive; *zinfo_or_arcname* is either the file
-   name it will be given in the archive, or a :class:`ZipInfo` instance.  If it's
-   an instance, at least the filename, date, and time must be given.  If it's a
-   name, the date and time is set to the current date and time.
+   Write a file into archive. The content is *data* which may be either a *str* or a *bytes* instance;
+   if it is a *str*, it is encoded as UTF-8.
+   *zinfo_or_arcname* is either the name of the file in the archive or a :class:`ZipInfo` instance.
```

Comment:
Please avoid unneeded rewrapping. Wrap only long lines that you have changed in any way.